### ISSUE TITLE
'Duplicate properties are not allowed within a block' with var error 

### ISSUE THREAD

**Issue by seeliang** [sasstools/sass-lint#907]
What version of Sass Lint are you using?
 1.9.1
Please include any relevant parts of your configuration

    "gulp-sass-lint": "1.2.0"

What did you do? Please include the actual source code causing the issue.
```scss  
  $paint-border-1: top;
  $paint-border-2: left;

  border: $size solid transparent;
  border-#{$paint-border-1}: $size solid $color;
  border-#{$paint-border-2}: $size solid $color;
```

What did you expect to happen?
no error

What actually happened? Please include any error messages given to you by Sass Lint.
'Duplicate properties are not allowed within a block'

If you're using a IDE plugin have you tried the CLI too?
Atom
